docs/AboutUs.md

```diff
@@ -19,15 +19,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Project Advisor
 
-### Jane Doe
+### Vaishak Anand
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/vaishakanand.png" width="200px">
 
-[[github](http://github.com/johndoe)]
+[[github](http://github.com/VaishakAnand)]
 [[portfolio](team/johndoe.md)]
```

Comment:
Might want to remove this line since everyone else also did so

Reply:
okay